py35/requirements-bin.txt

```diff
@@ -1,6 +1,5 @@
 anyjson-0.3.3-py3-none-any.whl
-# Basemap will be part of an extra package
-#basemap-1.0.8-cp35-cp35m-manylinux1_x86_64.whl
+basemap-1.0.8-cp35-cp35m-manylinux1_x86_64.whl
```

Comment:
Keep basemap commented out (with the related comment), as we discussed yesterday